### PR TITLE
Implementation of CrossEntropy and CrossEntropyBackprop as fused Op's

### PR DESCRIPTION
**Pull request by pruthviIntel**

### CODE REVIEW

**Comment by baojun-nervana** [comment]
Can you set the output element type? 
https://github.com/NervanaSystems/ngraph/commit/afd396dcf387119509b75928cf5ee38680265524
https://github.com/NervanaSystems/ngraph/blob/afd396dcf387119509b75928cf5ee38680265524/src/ngraph/op/fused/partial_slice.cpp#L231

File: src/ngraph/op/fused/crossentropy.cpp [hunk 1]
```diff
@@ -188,6 +188,23 @@ shared_ptr<Node> op::CrossEntropy::copy_with_new_args(const NodeVector& new_args
...
+    element::Type input_element_type = get_input_element_type(0);
+
+    NODE_VALIDATION_CHECK(this,
+                          input_element_type.is_dynamic() || input_element_type.is_real(),
+                          "Argument element type must be f16, bf16, f32, f64 or dynamic (got ",
+                          input_element_type,
+                          ").");
+    set_output_type(0, get_input_element_type(0), PartialShape::dynamic());
+
+    if (is_dynamic())
```
Comment (baojun-nervana) [inline src/ngraph/op/fused/crossentropy.cpp]: Is this necessary?

**Comment by pruthviIntel** [comment]
@jbobba can u review this?

**Comment by jbobba** [comment]
> @jbobba can u review this?

Looks good but please add the op to https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/op/fused_op_tbl.hpp

**Pull request merged**